packages/aws-cdk-lib/aws-efs/test/efs-file-system.test.ts

```diff
@@ -958,3 +958,162 @@ test.each([
     },
   });
 });
+
+describe('replication configuration', () => {
+  test('default settings', () => {
+    // WHEN
+    new FileSystem(stack, 'EfsFileSystem', {
+      vpc,
+      replicationConfiguration: {},
```

Comment:
Not sure about this, it provides an unclear API.
We should validate that either `destinationFileSystem` or `region` are defined, and update documentation and test accordingly.